scripts/dockerfiles/Dockerfile.release

```diff
@@ -20,6 +20,9 @@ COPY ["LICENSE", "NOTICE", "/"]
 
 COPY out/amazon-ecs-pause.tar /images/amazon-ecs-pause.tar
 
+# Copy our cni plugins ecs-eni, ecs-ipam and ecs-bridge
+COPY out/cni-plugins /cni-plugins
```

Comment:
I think that we should keep our plugins under `/cni-plugins` and not tie it very closely to only the `amazon-ecs-cni-plugins`. In cases where customers would want to build their own agent with custom cni-plugins, having a generic name is more meaningful imho. 

Also all our plugins are already prefixed with `ecs-`.